drivers/scheduler/k8s/systemtestSpec/sysbench/px-sysbench-app.yml

```diff
@@ -0,0 +1,72 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: sysbench-mysql-data
+spec:
+  storageClassName: sysbench-sc-secure
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Ti 
```

Comment:
Can we reduce the size from 1Ti to 512Gi as initial setup won't be having pool size 1Ti+